lib/schema/export.js

```diff
@@ -73,15 +73,13 @@ function encodeElementsInSchema(elements, schema) {
 }
 
 function encodeValueInSchema(value, schema) {
-  if (value instanceof OrValues) {
-    schema.oneOf = value.values.map(v => valueToSchemaTypeObject(v));
-  } else {
-    Object.assign(schema, valueToSchemaTypeObject(value));
-  }
+  Object.assign(schema, valueToSchemaTypeObject(value));
 }
 
 function valueToSchemaTypeObject(value) {
-  if (value instanceof QuantifiedValue) {
+  if (value instanceof OrValues) {
+      return {oneOf : value.values.map(v => valueToSchemaTypeObject(v))};
+  } else if (value instanceof QuantifiedValue) {
     if (value.isMaxUnbounded() || value.max > 1) {
       const ary = {
         type: 'array',
```

Comment:
I noticed that the `descendingFrom` and concept `label` features are defined in the extended schema, but you didn't actually implement them in the exporter.  Are you planning to do that next?